src/org/zaproxy/zap/extension/pscanrulesAlpha/files/licenses/xmpcore-license.txt

```diff
@@ -0,0 +1,38 @@
+The XMP Core library is distribued under terms of a modified BSD
+license.  The license is re-published in full below.
+
+@see http://mvnrepository.com/artifact/com.adobe.xmp/xmpcore/5.1.1
```

Comment:
Strange.  The vendor packaged the jar wrong.  META-INF says 5.1.1, but the sha1 is definitely the 5.1.2 version.